tests/components/comelit/test_diagnostics.py

```diff
@@ -0,0 +1,184 @@
+"""Tests for Comelit Simplehome diagnostics platform."""
+
+from __future__ import annotations
+
+from unittest.mock import patch
+
+from aiocomelit.const import (
+    BRIDGE,
+    CLIMATE,
+    COVER,
+    IRRIGATION,
+    LIGHT,
+    OTHER,
+    SCENARIO,
+    WATT,
+    AlarmAreaState,
+    AlarmZoneState,
+)
+
+from homeassistant.components.comelit.const import DOMAIN
+from homeassistant.components.comelit.diagnostics import TO_REDACT
+from homeassistant.components.diagnostics import REDACTED
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.const import CONF_TYPE
+from homeassistant.core import HomeAssistant
+
+from .const import (
+    BRIDGE_DEVICE_QUERY,
+    MOCK_USER_BRIDGE_DATA,
+    MOCK_USER_VEDO_DATA,
+    VEDO_DEVICE_QUERY,
+)
+
+from tests.common import MockConfigEntry
+from tests.components.diagnostics import get_diagnostics_for_config_entry
+from tests.typing import ClientSessionGenerator
+
+
+async def test_entry_diagnostics(
+    hass: HomeAssistant,
+    hass_client: ClientSessionGenerator,
+) -> None:
```

Comment:
Please use `SnapshotAssertion` in diagnostic tests

Reply:
Done!